ingest/ingest.py

```diff
@@ -39,8 +39,11 @@ def ingestRepos(repoESObjs):
 
         ghresponse = requests.get('https://api.github.com/repos/' + repo_name + '?access_token='+os.environ['GITHUB_ACCESS_TOKEN'], headers={'If-None-Match': repo_etag})
 
+        # gh_rate_limit_remaining = 5000 # Default quota for Github.
+        gh_rate_limit_remaining = ghresponse.headers['X-RateLimit-Remaining']
```

Comment:
Would maybe replace this with a ternary that defaults to 5000 if x-ratelimit-remaining isn't set, but it will probably never be unset.